tests/apps/grpcapp/app.go

```diff
@@ -0,0 +1,75 @@
+// ------------------------------------------------------------
```

Comment:
Please remove this grpcapp. You do not need this because e2e test is calling actual daprd grpc api. 

Please refer this code - https://github.com/dapr/dapr/blob/e92c3d468d94f4954eb45ad8333084fa5a5f831e/tests/apps/service_invocation/app.go#L622

https://github.com/dapr/dapr/blob/e92c3d468d94f4954eb45ad8333084fa5a5f831e/tests/apps/service_invocation/app.go#L658

You can see how you can create grpc client and call gRPC API. 

1. Create new handler inside tests/apps/stateapp/app.go
2. Implement handler, but this handler calls gRPC transaction api by referring service invocation app
3. add go test to tests/e2e/stateapp/stateapp_test.go to validate transactional api

Reply:
Added `grpc handler`

Reply:
Added [grpcHandler](https://github.com/dapr/dapr/pull/1681/files#diff-73b176cb473f99901f9ac0236d61410eR273)